notebooks/tutorials/flow_direction_and_accumulation/SRTMGL3_40.09_-105.56_40.209999999999994_-105.44.asc

```diff
@@ -0,0 +1,150 @@
+ncols        144
```

Comment:
Is this an output file generated by a notebook? If so, it shouldn't be under version control.

Reply:
No idea. Not related to the NST. This file doesn't exist in main landlab right now, so I deleted it.